Add documentation for Holiday integration (#29802)
home-assistant/home-assistant.github.io: source/_integrations/holiday.markdown

---
title: Holiday
description: Instructions on how to use holiday calendars in Home Assistant.
ha_category:
  - Calendar
ha_iot_class: Local Polling

> ha_release: 2023.12

Reviewer: ```suggestion
ha_release: 2024.1
```

ha_config_flow: true
ha_domain: holiday
ha_platforms:
  - calendar
ha_codeowners:
  - '@jrieger'
ha_integration_type: integration
---

> The Holiday integration allows you to create a calendar of holidays in Home Assistant for powering automations.

Reviewer: Probably we should ref the upstream library like we do in [holiday](https://www.home-assistant.io/integrations/workday/)

Author: Done.

A calendar entity has a state and attributes that represent the next upcoming event (only). A calendar trigger is a much more flexible way to power automations with fewer limitations than using the entity state.

{% include integrations/config_flow.md %}

> ## Calendar Event Automations

Reviewer: I suppose this section should/could give proposals on holiday calendar and not calendar automation in general (that could be a reference below)

Author: Done.

---
title: Holiday
description: Instructions on how to use holiday calendars in Home Assistant.
ha_category:
  - Calendar
ha_iot_class: Local Polling
ha_release: 2024.1
ha_config_flow: true
ha_domain: holiday
ha_platforms:
  - calendar
ha_codeowners:
  - '@jrieger'
ha_integration_type: integration
---

The Holiday integration allows you to create a calendar of holidays in Home Assistant for powering automations.

It uses the Python module [holidays](https://pypi.org/project/holidays/) to incorporate information about region-specific public holidays.

A calendar entity has a state and attributes that represent the next upcoming event (only). A calendar trigger is a much more flexible way to power automations with fewer limitations than using the entity state.

{% include integrations/config_flow.md %}

## Holiday calendar automations

Individual holiday *Events* are what powers automations such as:

> - Turn on the Christmas lights at the *start* of Christmas Day
> - Prevent your covers from being opened on public holidays
> 
> See [Calendar Automations](/integrations/calendar#automation) for an overview, and read more about [Calendar Trigger Variables](/docs/automation/templating/#calendar) for the available information you can use in a condition or action such as the event `summary`, `description`, `location` and more.

Reviewer 2: ```suggestion
- Turn on the Christmas lights at the *start* of Christmas Day.
- Prevent your covers from being opened on public holidays.

For an overview, refer to [calendar automations](/integrations/calendar#automation). You can use [calendar trigger variables](/docs/automation/templating/#calendar) in a condition or action such as the event `summary`, `description`, `location` and in other places.
```
